modules/profile/public/lib/queries/profiles/find.liquid

```diff
@@ -1,10 +1,10 @@
 {% liquid
-  if user_id == blank and id == blank and slug == blank and uuid == blank
+  if user_id == blank and id == blank and first_name == blank and last_name == blank and uuid == blank
```

Comment:
```suggestion
  if user_id == blank and id == blank and uuid == blank
```

I believe the idea here is to be able to use unique identifier to find a person - first name / last name are not unique identifiers :)